src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.cs

```diff
@@ -1168,9 +1168,11 @@ public static CultureInfo GetCultureInfo(string name, bool predefinedOnly)
 
             if (predefinedOnly && !GlobalizationMode.Invariant)
             {
-                return GlobalizationMode.UseNls ?
-                    NlsGetPredefinedCultureInfo(name) :
-                    IcuGetPredefinedCultureInfo(name);
+                if (GlobalizationMode.UseNls ? !CultureData.NlsIsEnsurePredefinedLocaleName(name): !CultureData.IcuIsEnsurePredefinedLocaleName(name))
```

Comment:
Is there any place calling NlsGetPredefinedCultureInfo or  IcuGetPredefinedCultureInfo? if not, please delete them. Thanks again for all your effort cleaning up this code.

Reply:
Hm, no I don't think so.